lib/WebpackConfig.js

```diff
@@ -51,6 +51,21 @@ class WebpackConfig {
         this.babelConfigurationCallback = function() {};
         this.useReact = false;
         this.loaders = [];
+        this.useTypeScriptLoader = false;
+        this.typeScriptOptions = {
+            transpileOnly: false,
+            happyPackMode: false,
+            logInfoToStdOut: false,
+            logLevel: 'info',
+            silent: false,
+            ignoreDiagnostics: [],
+            compiler: 'typescript',
+            configFileName: 'tsconfig.json',
+            visualStudioErrorFormat: false,
+            compilerOptions: {},
+            entryFileIsJs: false,
+            appendTsSuffixTo: []
+        };
```

Comment:
When we remove the validation below, we should be able to remove these, correct (since these are just the default values)?